### PR TITLE
feat(rush): add project suggestion

### PR DESCRIPTION
**Pull request by F3n67u**
add project suggestion support for rush spec

image

### CODE REVIEW

File: src/rush.ts [hunk 1]
```diff
@@ -1,40 +1,75 @@
...
+    const suggestions = [];
+
+    try {
+      if (out.trim() == "") {
+        return suggestions;
+      }
+
+      const rushConfigurationJson: IRushConfigurationJson = JSON.parse(
+        stripJsonComments(out)
+      );
```
Comment (fedeci) [inline src/rush.ts:23]: `out.trim()` does not mutate the original string:
```suggestion
      if (!out) {
        return suggestions;
      }

      const rushConfigurationJson: IRushConfigurationJson = JSON.parse(
        stripJsonComments(out.trim())
      );
```

**Review by fedeci — CHANGES_REQUESTED** [review]
A small nitpick but generally it looks good!

File: src/rush.ts (hunk 1 above)
Comment (F3n67u) [reply src/rush.ts]: fixed! thanks for review

**Review by fedeci — APPROVED** [review]
Thanks!

**Pull request merged**